conf/livy.conf

```diff
@@ -53,6 +53,24 @@
 # on user request and then livy server classpath automatically.
 # livy.repl.enableHiveContext =
 
+# Recovery mode of Livy. Possible values:
+# off: Default. Turn off recovery. Every time Livy shuts down, it stops and forgets all sessions.
+# recovery: Livy persists session info to the state store. When Livy restarts, it recovers
+#           previous sessions from the state store.
+# Must set livy.server.recovery.state-store and livy.server.recovery.state-store.url to
+# configure the state store.
+# livy.server.recovery.mode = off
```

Comment:
We are going to add failover support. There will be 3 possible values: off/recovery/failover.